examples/gaussian-ssm/script.jl

```diff
@@ -1,13 +1,75 @@
 # # Gaussian State Space Model
 using AdvancedPS
-using AbstractMCMC
+using Random
 using Distributions
-using Libtask: produce
-using StatsFuns
-
 using Plots
 
-x = [1, 2, 3]
-y = [2, 5, 1]
+# Parameters
+Parameters = @NamedTuple begin
+    a::Float64
+    q::Float64
+    r::Float64
+end
+
+a = 0.9   # Scale
+q = 0.32  # State variance
+r = 1     # Observation variance
+Tₘ = 300  # Number of observation
+Nₚ = 50   # Number of particles
+Nₛ = 1000 # Number of samples
+seed = 9  # Reproduce everything
+
+θ₀ = Parameters((a, q, r))
+
+mutable struct NonLinearTimeSeries <: AdvancedPS.AbstractStateSpaceModel
+    X::Vector{Float64}
+    θ::Parameters
+    NonLinearTimeSeries(θ::Parameters) = new(Vector{Float64}(), θ)
+    NonLinearTimeSeries() = new(Vector{Float64}(), θ₀)
+end
+
+f(m::NonLinearTimeSeries, state, t) = Normal(m.θ.a * state, m.θ.q) # Transition density
+g(m::NonLinearTimeSeries, state, t) = Normal(state, m.θ.r) # Observation density
+f₀(m::NonLinearTimeSeries) = Normal(0, m.θ.q / (1 - m.θ.a^2)) # Initial state density
+
+AdvancedPS.initialization(model::NonLinearTimeSeries) = f₀(model)
+AdvancedPS.transition(model::NonLinearTimeSeries, state, step) = f(model, state, step)
+function AdvancedPS.observation(model::NonLinearTimeSeries, state, step)
+    return logpdf(g(model, state, step), y[step])
```

Comment:
Is there a typo here? It does not seem `y` is in the arguments. 
```suggestion
    return logpdf(g(model, state, step), y[step])
```